src/pages/journal/components/test-outcome/test-outcome.actions.ts

```diff
@@ -1,9 +1,11 @@
 import { Action } from '@ngrx/store';
+import { TestSlot } from '../../../../shared/models/DJournal';
 
 export const TEST_OUTCOME_START_TEST = '[TestOutcomePage] Test Outcome start test';
```

Comment:
I know it's not been changed by you but can you fixe the naming as Test Outcome isn't a page

Reply:
Done.